### ISSUE TITLE
Issues with PostCSS

### ISSUE THREAD

**Issue by brunomiguel** [wangchucheng/hugo-eureka#147]
**Describe the bug**
When building the static content, PostCSS displays an error and doesn't build the CSS (even with the latest theme commit)

**To Reproduce**
run 'hugo'

**Expected behavior**

Builds the CSS

**Environment**

 - Hugo Version: 0.89.4
 - Eureka Version: 0.8.3 and 0.8.4

**Additional context**
The error:
```
Error: Error building site: POSTCSS: failed to transform "css/eureka.css" (text/css): Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib/node' is not defined by "exports" in /home/brunomiguel/projectos/fibrohell/fibrohell.github.io/themes/eureka/node_modules/postcss/package.json
    at new NodeError (node:internal/errors:371:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:429:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:683:3)
    at resolveExports (node:internal/modules/cjs/loader:482:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/brunomiguel/projectos/fibrohell/fibrohell.github.io/themes/eureka/node_modules/tailwindcss/lib/util/processPlugins.js:12:36) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
```

**Comment by brunomiguel** [comment]
It was an issue with a globally installed npm module. It's fixed now

**Comment by vk0xOrg** [comment]
I have a similar issue. If it's not related sorry for necrobumping, then will make another issue.
So, while trying to build static site the following error arise:

`Error: Error building site: POSTCSS: failed to transform "css/eureka.css" (text/css): resource "css/project1/css/eureka.css_fc3f76d7bee2760c3a903059afc3d9b2" not found in file cache
`

I've found that if  **baseURL** is set to "**/**", then everything works as expected; if it is set to "**https://user.github.io/project1**" then the build fails and displays the above error. 

Tried `npm i -g postcss-cli autoprefixer` locally as well as a GitHub workflow in actions but neither worked. 

What else could be done?
